src/library_async.js

```diff
@@ -671,7 +672,14 @@ mergeInto(LibraryManager.library, {
 #if ASYNCIFY_DEBUG
           err('ASYNCIFY: start: ' + start);
 #endif
-          Module['asm'][start]();
+          var asyncReturnValue = Module['asm'][start]();
+          if (Asyncify.currData === null) {
```

Comment:
`(Asyncify.currData === null)` means that no asynchronous execution is running.

Reply:
In other places we just do `!Asyncify.currData` which is shorter.

Reply:
Sounds good to me, I'll update that. I was worried about whether `currData` could be 0 since it is an integer type, but I suppose it is probably never 0 as it is a stack pointer.